boards/nrf52840dk_nrf52840.overlay

```diff
@@ -3,6 +3,13 @@
  * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
  */
 
+/ {
+	chosen {
```

Comment:
If the sample is going to support OTA DFU on the nRF5340 DK, it needs the below code in its overlay as well.

Reply:
I have not looked at OTA at all.